adobecom/helpx-internal
# Configure project to add library plugin in sidekick

Implementation of the Milo Library plugin for the Franklin Sidekick,
details have been documented in https://github.com/adobecom/helpx-internal/blob/configuration-sidekick-library/docs/sidekick.md

If you want to test, you need to:
* modify the helpx-internal configuration in the Sidekick (in your browser) and make the repository point to https://github.com/adobecom/helpx-internal/tree/configuration-sidekick-library
* save it
* load a Word document from the Helpx Internal Sharepoint
* load the Sidekick
* click the `Library` button
* in the bottom left dialog, click `Blocks`
=> You should see the `Note` accordion with 3 samples you can copy paste.
